### ISSUE TITLE
Installing dep mac / node v4.9.1

### ISSUE THREAD

**Issue by reggi** [kesla/node-snappy#183]
I'm trying to build a module that uses snappy on node v4.9.1

I'm getting two issues:

1) Restructuring the options
2) fatal error: 'algorithm' file not found

```
> snappy@6.2.3 install /private/node_modules/snappy
> prebuild-install || node-gyp rebuild

/private/node_modules/snappy/node_modules/prebuild-install/node_modules/simple-get/index.js:18
    const { hostname, port, protocol, auth, path } = url.parse(opts.url) // eslint-disable-line node/no-deprecated-api
          ^

SyntaxError: Unexpected token {
    at exports.runInThisContext (vm.js:53:16)
    at Module._compile (module.js:373:25)
    at Object.Module._extensions..js (module.js:416:10)
    at Module.load (module.js:343:32)
    at Function.Module._load (module.js:300:12)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (/private/node_modules/snappy/node_modules/prebuild-install/download.js:3:11)
    at Module._compile (module.js:409:26)
    at Object.Module._extensions..js (module.js:416:10)
  CXX(target) Release/obj.target/snappy/deps/snappy/snappy-1.1.7/snappy-sinksource.o
clang: warning: include path for libstdc++ headers not found; pass '-stdlib=libc++' on the command line to use the libc++ standard library instead [-Wstdlibcxx-not-found]
  CXX(target) Release/obj.target/snappy/deps/snappy/snappy-1.1.7/snappy-stubs-internal.o
clang: warning: include path for libstdc++ headers not found; pass '-stdlib=libc++' on the command line to use the libc++ standard library instead [-Wstdlibcxx-not-found]
../deps/snappy/snappy-1.1.7/snappy-stubs-internal.cc:29:10: fatal error: 'algorithm' file not found
#include <algorithm>
         ^~~~~~~~~~~
1 error generated.
make: *** [Release/obj.target/snappy/deps/snappy/snappy-1.1.7/snappy-stubs-internal.o] Error 1
gyp ERR! build error 
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/Users/tom/.nvm/versions/node/v4.9.1/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:276:23)
gyp ERR! stack     at emitTwo (events.js:87:13)
gyp ERR! stack     at ChildProcess.emit (events.js:172:7)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:211:12)
gyp ERR! System Darwin 19.4.0
command "/Users/tom/.nvm/versions/node/v4.9.1/bin/node" "/Users/tom/.nvm/versions/node/v4.9.1/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
```

Any ideas on how to fix these issues?

**Comment by reggi** [comment]
related: https://github.com/kesla/node-snappy/issues/77

**Comment by kesla** [comment]
@reggi This is fixed now, right, in #184
